services/kubernetes-dashboard/metadata.yaml

```diff
@@ -2,6 +2,8 @@ displayName: Kubernetes Dashboard
 description: General purpose, web-based UI for Kubernetes clusters. It allows users to manage applications running in a cluster and troubleshoot them, as well as manage the cluster itself.
 category:
   - monitoring
+dependencies:
+  - traefik
```

Comment:
In the UI, we already require all "foundational" apps to be deployed. Traefik is one of the foundational apps. 

would the foundational app logic already handle telling users about this dependency?